source/accounts/role_accounts.rst

```diff
@@ -0,0 +1,99 @@
+
+.. _role_accounts:
+
+Role Accounts
+=============
+
+A role account is a user account shared with one or more users as
+members, such that all group members share the account equally via the
+use of `sudo`, typically for the unified use of a project. Role
+accounts must have a name that reflects their project association or
+function. All changes to a role account must be submitted by the role
+account Principal Investigator (PI) via OTRS ticket. Send email to
+rdhpcs.aim.help@noaa.gov with the subject line: Role account changes -
+first.last (role account name).
+
+Accessing a Role Account
+------------------------
+
+You can access any role account if you are a member of the account.
+Use sudo with your RSA token to gain access. For example:
+
+ .. code-block:: shell
+
+   jsmith# sudo su - roleuser
+   Access is via First.Last username only. Enter RSA PASSCODE:
+   bash-4.1$ whoami
+   roleuser
+
+Your RSA passcode is your PIN+Token code.
+
+X Applications With Role Accounts
+---------------------------------
+
+If you are planning to use X utilities with role accounts, you should
+use the xsudo utility to switch to the role account instead of using
+the "sudo" command directly. You need to explicitly set the DISPLAY
+environment variable after doing the xsudo to the role account. So for
+example, if you want to use role.rap-chem role account and would like
```

Comment:
Is `rap-chem` a real account?

Reply:
I hope not. It's supposed to be made up to slow down the people who blindly cut-n-paste